README.md

```diff
@@ -38,45 +42,52 @@ You can access DOM elements with javascript.
 </script>
 ```
 
-`document.getElementsByClassName("myClass")` returns an [_array-like object_](http://www.nfriedly.com/techblog/2009/06/advanced-javascript-objects-arrays-and-array-like-objects/) of all elements with the class "myClass";
+`document.getElementsByClassName("myClass")` returns a live [HTML Collection](https://developer.mozilla.org/en-US/docs/Web/API/HTMLCollection) of all elements with the class "myClass";
 
 ```html
-<li class="menu-item">
-  London
-</li>
-<li class="menu-item">
-  Nazareth
+<li class="menu-item">London
 </li>
+<li class="menu-item">Nazareth</li>
 <script>
   var menuItems = document.getElementsByClassName("menu-item");
-  // [_li.menu-item_,_li.menu-item_]
+  // [li.menu-item, li.menu-item]
 </script>
 ```
 
-`document.querySelector(myCssSelector)` returns the first element matching `myCssSelector`, where `myCssSelector` takes the form of a [CSS selector](https://developer.mozilla.org/en-US/docs/Web/CSS/CSS_Selectors) for example "#myId", ".myClass", "myTag", etc etc.
+It's important to note that this collection is _live_. If you change one of the elements within it, then try to access the collection again it will be updated. This is different to how most function return values work, so be careful.
```

Comment:
Great note. Also, the second workshop of this day is about pure functions. Should we add a comment in brackets that this will be covered in the next wp about pure functions?

Reply:
I think it might just confuse them to talk about upcoming stuff when they're trying to focus on this